nitric/api/storage.py

```diff
@@ -91,25 +92,34 @@ async def write(self, body: bytes):
         Will create the file if it doesn't already exist.
         """
         try:
-            await self._storage._storage_stub.write(storage_write_request=StorageWriteRequest(bucket_name=self._bucket, key=self.key, body=body))
+            await self._storage._storage_stub.write(
+                storage_write_request=StorageWriteRequest(bucket_name=self._bucket, key=self.key, body=body))
         except GRPCError as grpc_err:
             raise exception_from_grpc_error(grpc_err)
 
     async def read(self) -> bytes:
         """Read this files contents from the bucket."""
         try:
-            response = await self._storage._storage_stub.read(storage_read_request=StorageReadRequest(bucket_name=self._bucket, key=self.key))
+            response = await self._storage._storage_stub.read(
+                storage_read_request=StorageReadRequest(bucket_name=self._bucket, key=self.key))
             return response.body
         except GRPCError as grpc_err:
             raise exception_from_grpc_error(grpc_err)
 
     async def delete(self):
         """Delete this file from the bucket."""
         try:
-            await self._storage._storage_stub.delete(storage_delete_request=StorageDeleteRequest(bucket_name=self._bucket, key=self.key))
+            await self._storage._storage_stub.delete(
+                storage_delete_request=StorageDeleteRequest(bucket_name=self._bucket, key=self.key))
         except GRPCError as grpc_err:
             raise exception_from_grpc_error(grpc_err)
 
+    async def upload_url(self, expiry: int = 3600):
```

Comment:
The Node SDK defaults to 600 seconds for the expiry. Is this something we should be keeping consistent across SDKs?

```suggestion
    async def upload_url(self, expiry: int = 600):
```